templates/combined/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+resources:
+- nfd-daemonset-combined.yaml
+- nfd-worker-conf.yaml
```

Comment:
this is OK to me but it could also be something like:

```
configMapGenerator:
- name: nfd-worker-config
  namespace: node-feature-discovery
  files:
  - nfd-worker.conf
```

Reply:
https://github.com/kubernetes-sigs/node-feature-discovery/pull/558/commits/8b9f5a8715229f976e9566b12b055fb3769a0156